contributors/design-proposals/api-machinery/aggregated-api-servers.md

```diff
@@ -210,7 +210,7 @@ API for all these operations.
 
 While deleting a namespace, kubernetes server needs to ensure that resources in
 that namespace maintained by other servers are deleted as well. We can do this
-using resource [finalizers](namespaces.md#finalizers). Each server
+using resource [finalizers](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/namespaces.md#finalizers). Each server
```

Comment:
You may want to use this instead of the full URL: `/contributors/design-proposals/architecture/namespaces.md#finalizers`